assets/scripts/gallery/GalleryContents.jsx

```diff
@@ -117,7 +116,10 @@ function GalleryContents (props) {
               selected={selectedStreet === item.id}
               doSelect={selectStreet}
               doDelete={deleteStreet}
-              showStreetOwner={userId !== item.creatorId}
+              showStreetOwner={
+                !userId ||
+                !(userId === item.creatorId || userId === item.creator_id)
```

Comment:
If we're changing `creatorId` to `creator_id` in what cases will `creatorId` still be the property that is present?

Reply:
Have resolved, `creatorId`(and snakeCase in general) will be used everywhere in code.